app/scripts/modules/core/src/deploymentStrategy/strategies/rollingredblack/additionalFields.html

```diff
@@ -1,9 +1,21 @@
-<div class="form-group">
+<div class="form-group" style="margin-top: 15px">
   <div class="col-md-6">
     <label>
       Percentages
       <help-field key="strategy.rollingRedBlack.targetPercentages"></help-field>
     </label>
     <number-list model="$ctrl.command.targetPercentages" label="percentage"/>
   </div>
+  <div class="col-md-12 form-inline">
+    <label>Wait Before Disable
```

Comment:
This will possibly change to `Wait Before Termination` depending on the outcome of a couple conversations around RRB.